test/net2cog/environment.yaml

```diff
@@ -10,6 +10,6 @@ dependencies:
   - rio-cogeo=5.3.4
   - numpy=1.24.3
   - pip=23.1.2
-  - matplotlib=3.10.3
+  - matplotlib=3.9.1
```

Comment:
The test instructions on the original PR mentioned running things building an environment with `conda` instead of Micromamba. I think that's how this slipped through. Interesting that things worked with one and not the other.